app.py

```diff
@@ -168,13 +168,16 @@ def homepage():
     return flask.render_template('index.html')
 
 
+@app.route('/discover')
+def discover():
+    return flask.render_template('discover.html')
+
+
 @app.route('/search')
 def search_snap():
     snap_searched = flask.request.args.get('q', default='', type=str)
     if(not snap_searched):
-        return flask.render_template(
-            'search.html'
-        )
+        return flask.redirect('/discover')
```

Comment:
@tbille @nottrobin Greg asked if we could redirect search with empty query to discover page. Do you think it's ok to do it this way?

Reply:
It's okay for me

Reply:
yeah okay